backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/action/ImagesActionsParametersBase.java

```diff
@@ -19,6 +20,7 @@ public class ImagesActionsParametersBase extends StorageDomainParametersBase {
     private boolean importEntity;
     private boolean leaveLocked;
     private Long initialSizeInBytes;
+    private DiskBackup backup;
```

Comment:
I would suggest to rename to `diskBackup` (data member, getter, setter & getter's parameter).

Reply:
With this factoring, the ConvertDiskCommandParameters will be effected as well:

method ImagesActionsParametersBase.getBackup() > getDiskBackup
method ImagesActionsParametersBase.setBackup(DiskBackup) > setDiskBackup
method ConvertDiskCommandParameters.getBackup() > getDiskBackup
method ConvertDiskCommandParameters.setBackup(DiskBackup) > setDiskBackup

Reply:
this was always named backup, including the API 
https://github.com/oVirt/ovirt-engine/blob/bba3bb6ad58e461e390aa8e110d888a409d4a7bd/backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/storage/BaseDisk.java#L116

so I see no value in adding the prefix

Reply:
Resolving conversation

Reply:
Ack.